.eslintrc.json

```diff
@@ -29,6 +42,12 @@
       "rules": {
         "no-console": "off"
       }
+    },
+    {
+      "files": "**/__fixtures__/**/*",
```

Comment:
covered by the override above

Reply:
ah didn't noticed when I rebased, removed